Justfile

```diff
@@ -189,27 +189,30 @@ test-unit *ARGS: devenv
     $BIN/python -m pytest tests/unit {{ ARGS }}
     $BIN/python -m pytest --doctest-modules ehrql
 
-# Run the generative tests only. Optional args are passed to pytest.
+# Run the generative tests only, configured to use more than the tiny default
+# number of examples. Optional args are passed to pytest.
 #
 # Set GENTEST_DEBUG env var to see stats.
 # Set GENTEST_EXAMPLES to change the number of examples generated.
+# Set GENTEST_MAX_DEPTH to change the depth of generated query trees.
 test-generative *ARGS: devenv
-    $BIN/python -m pytest tests/generative {{ ARGS }}
+    GENTEST_EXAMPLES=${GENTEST_EXAMPLES:-200} \
+    GENTEST_RANDOMIZE=${GENTEST_RANDOMIZE:t} \
```

Comment:
This one isn't documented (there's a mention of randomizing with `just test-generative`, and it's unlikely to be something we want to turn on and off locally, but since it can be specified as an env var, I think it'd be worth adding it to the developer docs